opm/material/fluidsystems/blackoilpvt/WetGasPvt.hpp

```diff
@@ -706,7 +704,7 @@ class WetGasPvt
 private:
     void updateSaturationPressure_(unsigned regionIdx)
     {
-        typedef std::pair<Scalar, Scalar> Pair;
+        using Pair = std::pair<Scalar, Scalar>;
```

Comment:
Can't help it as the regular person for this is not here :wink:.
Do we even need this? What about just using this below
```
-            Pair val(Rv, pSat);
-            pSatSamplePoints.push_back(val);
+            pSatSamplePoints.emplace_back(Rv, pSat);
```

Reply:
okay. Pair is also used in lambda parameters below, so i changed to a variadic lambda there.